src/shipit_code_coverage/shipit_code_coverage/taskcluster.py

```diff
@@ -20,14 +21,16 @@ def _get_build_platform_name(platform):
 
 
 def get_last_task(platform):
-    r = requests.get(index_base + 'task/gecko.v2.mozilla-central.latest.firefox.{}'.format(_get_build_platform_name(platform)))
+    index = get_service('index')
```

Comment:
Since we need the index and queue services all the time, I'd set them up at the top of the file so you don't have to define them in each function.

Reply:
@marco-c that will make it a global variable, 'index' is a generic variable name , I was worried about any naming conflicts during import. Refactoring the code into a class and  making it as 'self.index' was another option. Your thoughts ?

Reply:
You can call it `index_service` to avoid naming conflicts. I'd avoid refactoring the code into a class, as it's possible this module will be come so small that we could totally remove it.